kairon/shared/data/action_serializer.py

```
@@ -0,0 +1,467 @@
+                           v.required and k not in {'bot', 'user', 'timestamp', 'status'}}
+
+        for action in actions_list:
+            if not isinstance(action, dict):
+                err_summary.append(f"Expected dictionary for [{action_type}] ")
+                continue
+            action_name = ActionSerializer.get_item_name(action, raise_exception=False)
+            if not action_name:
+                err_summary.append(f"No name found for  [{action_type}].")
+                continue
+            if action_name in encountered_action_names:
+                if action_type != ActionType.form_validation_action.value and not action_name.startswith("utter_"):
```

Ruff

Use a single `if` statement instead of nested `if` statements

(SIM102)